include/ck_tile/core/tensor/slice_tile.hpp

```diff
@@ -86,7 +86,8 @@ set_slice_tile(static_distributed_tensor<DstDataType_, DstStaticTileDistribution
 
     static_assert(std::is_same_v<decltype(sliced_dstr), DstDistribution>, "wrong!");
```

Comment:
sliced_dstr should be same as `SrcDistribution`.   Check [PR 2232](https://github.com/ROCm/composable_kernel/pull/2232)